docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -176,3 +177,30 @@ For the remaining users:
 
 .. literalinclude:: resource-groups-example.json
     :language: json
+
+.. _scheduleweight-ex:
+
+``schedulingWeight`` example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Schedule weighting is a method of assigning a priority to a resource. Higher
+priority resources are assigned a higher scheduling weight in order to ensure
```

Comment:
Should be the reverse order, because the user sets the weight while treating the resource as higher priority is what Trino does. So:

"Subgroups with a higher scheduling weight are given higher priority in order to ensure..."